sdk/include/psdk/shellapi.h

```diff
@@ -618,6 +618,12 @@ SHEmptyRecycleBinW(
   _In_opt_ LPCWSTR pszRootPath,
   DWORD dwFlags);
 
+BOOL
+WINAPI
+SHRunControlPanel(
+  _In_ LPCWSTR commandLine,
+  _In_opt_ HWND parent);
```

Comment:
```suggestion
BOOL
WINAPI
SHRunControlPanel(
  _In_ LPCWSTR lpcszCmdLine,
  _In_opt_ HWND hwndMsgParent);
```
And this should be in shlobj.h instead.

Reply:
Finally not, in order to keep the same prototype as WINE

Reply:
(unless i'm wrong)

Reply:
> same prototype as WINE

Your commits are going back and forth: that does not make sense :-(
Either all this is sync'ed with Wine (which I believe it is not) or Windows: your proposal matches neither...

Reply:
Please see my last message : For the prototype, I mixed you comments and the ones from Hermes. Is the following ok :
In LPCWSTR commandLine, In_opt HWND parent ?

Reply:
Aligned with WINE : https://source.winehq.org/git/wine.git/blob_plain/HEAD:/dlls/shell32/shellord.c